### DISCUSSION TITLE
Project Status Updates

### DISCUSSION THREAD

**Discussion by SteffeyDev** [danielbuechele/atemOSC#236] [General]
Sorry updates have been slow.  My current testing environment is only available during the week, which means I have to use PTO or work at night to test the app.  That has been hard to justify lately, but I do really want to get the next version tested and released.  I'm hoping to get a more regular testing and release schedule established, but as one of many side projects I can't always be as consistent with this as I want to.

**Comment by randallpacker** [comment]
Peter, just wanted to let you know how much I appreciate the incredible work you are doing! I have two ATEM HD Switchers and 3 Hyperdecks, so please let me know if there is anything I can do to help in terms of testing and design. 

Randall
